### ISSUE TITLE
Showing different NFT after minitng

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#10060]
Its bit dissapointing that after minting I dont see my NFT but I see the preview picture for collection. It may be confusing for users

https://github.com/kodadot/nft-gallery/assets/104380776/20fa2200-69ac-4186-bd45-bb368b1e9999

**Comment by prury** [comment]
it takes some time for the nft to be updated Lu, 8 seems the same as your image for me now

**Comment by JustLuuuu** [comment]
yeah, I know that, but the user doesn't know; Im just not sure if showing a picture of the collection is a better option, but maybe yeah.
